ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -180,6 +180,22 @@ def min(*input, dim=None, keepdim=False, out=None):
         )
 
 
+@numpy_to_torch_style_args
+@to_ivy_arrays_and_back
+def mode(*input, dim=None, keepdim=False, out=None):
```

Comment:
the function signature should be `input, dim=- 1, keepdim=False, *, out=None`
ref: https://pytorch.org/docs/stable/generated/torch.mode.html